database/src/main/postgres/public/V1.2.2__jsonb_array_to_text_array.sql

```diff
@@ -30,10 +29,10 @@ CREATE OR REPLACE FUNCTION public.jsonb_array_to_text_array(
 --      Postgres array
 --
 -------------------------------------------------------------------------------
-    LANGUAGE sql IMMUTABLE STRICT PARALLEL SAFE
-BEGIN ATOMIC
-    SELECT ARRAY(SELECT jsonb_array_elements_text(i_json_array));
-END;
-
+LANGUAGE sql IMMUTABLE STRICT PARALLEL SAFE
+AS
+    $$
+        SELECT ARRAY(SELECT jsonb_array_elements_text(i_json_array));
+    $$;
```

Comment:
If I understood it correctly, I would keep the original actually:
* https://dba.stackexchange.com/questions/300507/what-does-begin-atomic-mean-in-a-postgresql-sql-function-procedure/300512#300512
* https://stackoverflow.com/questions/12144284/what-are-used-for-in-pl-pgsql/12172353#12172353
* see the `sql_body` segment: https://www.postgresql.org/docs/14/sql-createfunction.html

Basically, this is a SQL function and this `BEGIN ATOMIC` is new syntax available from PG 14+ and it's doing parsing at function definition time, not runtime as with your changes.

This is similar to writing the text of the function body as a string constant (see definition above), but there are some differences: This form only works for LANGUAGE SQL, the string constant form works for all languages. This form is parsed at function definition time, the string constant form is parsed at execution time; therefore this form cannot support polymorphic argument types and other constructs that are not resolvable at function definition time.

Reply:
Let me explain. I have changed it to this version so that I can use the Flyway plugin. I can see that it might have not been the best idea, after all I could keep the original one and add use this one just for the purposes of local testing. The thing is that Flyway works only with Postgres up to version 13. It fails with the original version of the function.

Would that be better if I keep both for now? By the way I hope we will soon be able to somehow automate this db stuff with Ultet. If I understood correctly it should be able to do such things.

Reply:
Okay, understood, in that case..given how trivial this is & it's not like there are many dependencies for this, I guess it's okay if the parsing is not during function definition time. We can create a ticket for this one to get it back.

Reply:
I have added the file back in its original form. And added additional one just for the local testing.